rules/windows/builtin/win_invoke_obfuscation_var+_services.yml

```diff
@@ -0,0 +1,43 @@
+action: global
+title: Invoke-Obfuscation VAR+ Launcher
+id: 8ca7004b-e620-4ecb-870e-86129b5b8e75
+description: Detects Obfuscated use of Environment Variables to execute PowerShell
+status: experimental
+author: Jonathan Cheong, oscd.community
+date: 2020/10/15
+references:
+     - https://github.com/Neo23x0/sigma/issues/1009 #(Task 24)
+tags:
+    - attack.defense_evasion
+    - attack.t1027
+    - attack.execution
+    - attack.t1059.001
+falsepositives:
+    - Unknown
+level: high
+detection:
+    selection_1:
+        - ImagePath|re: 'set\s[a-zA-Z]{3,6}=Invoke-Expression'
+        - ImagePath|re: '(\"(?:\{\d\}){1,7}\\){1,5}'
+    condition: selection and selection_1
+---
+logsource:
+    product: windows
+    service: system
+detection:
+    selection:
+        EventID: 7045
+---
+logsource:
+    product: windows
+    service: sysmon
+detection:
+    selection:
+        EventID: 6
+---
+ logsource:
+     product: windows
+     service: security
+ detection:
+     selection:
+         EventID: 4697
```

Comment:
These log sources are defined but not used anywhere

Reply:
Just to correct what I said here, this would only select one of the selections, as the key is triplicated

Reply:
Hey @w0rk3r, I was basing my rule off this [existing rule](https://github.com/Neo23x0/sigma/blob/master/rules/windows/builtin/win_invoke_obfuscation_obfuscated_iex_services.yml) which was referenced as an example of what needed to be created per [Task ](https://github.com/Neo23x0/sigma/issues/1009)

That rule is in the sigma oscd branch, so I assumed it's gone through the checks and is valid - is it not?

Sorry if it's got an obvious answer - I'm new to Sigma.

Reply:
I could nest the selection > EventID values in the same detection, but my issue is there're three logsource values. Is it possible for a Sigma rule to have multiple log sources and a detection condition that draws on all of them? Or will I need to create 3 separate rules for each log source?

I've looked through a few existing rules in the project and so far haven't found any examples that allow for a detection condition that works with multiple logsources.